src/ddmd/mtype.d

```diff
@@ -910,6 +910,18 @@ extern (C++) abstract class Type : RootObject
         return buf.extractString();
     }
 
+    static const(char*)[2] toAutoQualChars(Type t1, Type t2)
```

Comment:
Please add DDoc comment header.

Reply:
Also, why the word "Auto"?  I would have expected something like `toFullyQualChars`.

Reply:
Will do. Auto is because it only qualifies the types if they would otherwise collide. Suggestions welcome.

Reply:
Ok, I see.  No need to change it in my opinion.  I was just curios.